.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: node_js
 node_js:
-  - '6'
-  - '4'
+  - node
+  - lts/12
+  - lts/10
```

Comment:
Travis didn't run for this  PR, maybe just ?
```
- 12
- 10
```

Reply:
That's fine.

Reply:
https://travis-ci.org/github/kevva/to-ico/builds/667045841

Reply:
Sorry, my bad.